subcommand/webhook-cert-manager/command_test.go

```diff
@@ -369,19 +369,16 @@ func TestCertWatcher(t *testing.T) {
 	defer stopCommand(t, &cmd, exitCh)
 
 	ctx := context.Background()
-	// This first sleep is required to ensure the command had completed validating that the MWC does exist in the cluster
-	// It ensures the MWC is absent when the server actually attempts to update it with a bundle.
-	time.Sleep(1 * time.Millisecond)
+	// This sleep ensures the webhook has been updated and the reconcile has been successful.
```

Comment:
Can you instead loop and check that the mwc has its caCert len > 0? That will remove race conditions.

Reply:
You need to loop on a `Get` to ensure the first iteration has run through and updated the MWC I think.

Reply:
done 😄